src/ctapipe/containers.py

```diff
@@ -1189,6 +1189,18 @@ class MuonParametersContainer(Container):
         nan * u.deg**2,
         "MSE of the deviation of all pixels after cleaning from the ring fit",
     )
+    ring_size = Field(nan, "Number of pixels in the ring")
```

Comment:
Is this really number of pixels? The code below computes some of the pixel intensities.

If this is actually the sum of pixel charges, the name should be `ring_intensity` and the help string match what is actually computed.